base_layer/wallet/src/ffi.rs

```diff
@@ -178,7 +219,8 @@ pub unsafe extern "C" fn add_transaction_input(
     maturity: c_ulonglong,     // u64
 ) -> bool
 {
-    // append input to tx
+    //(*transaction).addInput();
```

Comment:
This will obviously be replaced but I just want to note that in the Tari codebase we are using snake case for method names and CamelCase for Type names.